Sources/WalletConnectNetworking/HTTPClient/HTTPService.swift

```diff
@@ -12,6 +12,7 @@ public protocol HTTPService {
     var method: HTTPMethod { get }
     var body: Data? { get }
     var queryParameters: [String: String]? { get }
+    var headerFields: [String: String]? { get }
```

Comment:
may be `additionalHeaders`, because it's not represent all request headers as we are setting `Content-Type` separately after